packages/contracts-bedrock/src/interop/CrossL2Inbox.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ISemver } from "src/universal/ISemver.sol";
+
+/// @notice Entry to post to the inbox.
+///         The postie may deliver multiple entries per mail delivery.
+/// @custom:field chain   Chain identifier.
+/// @custom:field output  Output-root of the chain.
+struct InboxEntry {
+    bytes32 chain;
+    bytes32 output;
+}
+
+/// @custom:proxied
+/// @title CrossL2Inbox
+/// @notice The CrossL2Inbox receives output-roots of any chain,
+///         and makes the output-roots available for cross-L2 proving.
+contract CrossL2Inbox is ISemver {
+    /// @notice The address that is allowed to post into the inbox.
+    /// This is temporary for Interop Milestone 0:
+    /// this will be changed to a system-only address later.
+    address internal immutable SUPERCHAIN_POSTIE;
```

Comment:
typo?

Reply:
Postie vs poster?

Reply:
No, it's aussie slang for postman, was looking for a gender neutral name that's not as long :sweat_smile:

Reply:
I see its supposed to be postie, i just never really saw that word before. Can ignore my comments